pal/inc/pal_mstypes.h

```diff
@@ -200,7 +200,7 @@ extern "C" {
 // they must be either signed or unsigned) and we want to be able to use 
 // __int64 as though it were intrinsic
 
-#ifdef BIT64
+#ifndef BIT64
```

Comment:
> Is there any particular reason for not using typedef int64_t __int64 instead of all these BIT64 stuff ?

Don't know of any particular reason, we just copied this and other files as is from dotnet/coreclr to reuse existing work.